cookbook/assetic/uglifyjs.rst

```diff
@@ -231,8 +232,14 @@ the node package is installed:
 
 .. code-block:: bash
 
+    # global installation
     $ npm install -g uglifycss
 
+    # local installation
+    $ cd /path/to/your/symfony/project
+    $ mkdir app/Resources/node_modules
```

Comment:
Is this step needed? Doesn't `npm` create the directory itself if it does not exist?